blueprints/app/files/tests/helpers/module-for-acceptance.js

```diff
@@ -11,12 +11,12 @@ export default function(name, options = {}) {
       this.application = startApp();
 
       if (options.beforeEach) {
-        return options.beforeEach.apply(this, arguments);
+        return options.beforeEach.apply(...arguments);
```

Comment:
This is wrong actually.  It is a bug in JSCS's rule.  If we switch from `options.beforeEach.apply(this, arguments);` to `options.beforeEach(...arguments);` the `this` context inside of `options.beforeEach` changes from the same thing as the current context to the POJO itself (which is incorrect).